packages/config-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@redhat-cloud-services/frontend-components-config-utilities",
-    "version": "1.5.31",
+    "version": "2.0.0-beta.1",
```

Comment:
Any reason for the beta tag name? It will be overwritten by the release script anyway.

Reply:
I think I will do a manual major release. I have used these to create the same beta releases for testing.